src/deploy/table_account_block.sql

```diff
@@ -0,0 +1,22 @@
+-- Deploy maevsi:table_account_block to pg
+-- requires: schema_public
+-- requires: table_account_public
+
+BEGIN;
+
+CREATE TABLE maevsi.account_block (
+  id                  UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+  author_account_id   UUID NOT NULL REFERENCES maevsi.account(id),
+  blocked_account_id  UUID NOT NULL REFERENCES maevsi.account(id),
+  created             TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  UNIQUE (author_account_id, blocked_account_id),
+  CHECK (author_account_id != blocked_account_id)
+);
+
+COMMENT ON TABLE maevsi.account_block IS 'Blocking of an account by another account.';
```

Comment:
Add smart tags there so that no update or delete api endpoints will be generated by postgraphile. You can remove `,update` from the line below then too.

Reply:
Woud it be possible to add `@omit update,delete` to the table comment and remove `@omit` from the column comment completely? I am asking because elsewhere `@omit` seems to always be used in a column comment. Is there a reason for this?

Reply:
Yes, that's possible, but keep the the `@omit` for `create` on the on the id column as it's still possible to create entries in that table for which the id should not be set manually.

Reply:
The table `legal_term` now also has an `@omit` on the table level. You can see combinations and meanings for that here: https://www.graphile.org/postgraphile/smart-tags/#omit